Add projects and about section to portafolio template

rega1237 opened the pull request:
# In this Branch Added
- All the HTML structure for the Recent Projects and About of the personal portfolio template number 3.
- Add the style to make it look as close as possible to the design presented in Figma.
- Add all the images like social icons and background image
- Add the respective linters to check the code
- Update the README file

This Project made to begin to build my portfolio.

rega1237 committed:
Add section "recent-projects"

Add section recent-projects to the index file and add the title of the section.

rega1237 committed:
Update styles

Add grid display to the recent-projects section and add style to the recent-work-title.

rega1237 committed:
Add card project

Add HTML content of the first card and style the card box

rega1237 committed:
Add style card box

Add style to all the elements inside the card project

rega1237 committed:
Add other cards

Add all the others cards in the template.

rega1237 committed:
Add about section

Add about section and the text content with the button get my resume

rega1237 committed:
Update styles file

Add style to the intro section in about me.

rega1237 committed:
add images

Add images for about me background section

rega1237 committed:
Add Languages card

Add languages card to index file

rega1237 committed:
Update styles file

Add style to the languages card and make the section where all the cards gonna be placed as a grid.

rega1237 committed:
Add other about cards

Add styled skills card and frameworks card.

rega1237 committed:
Updated README

Update Readme of the project.

rega1237 committed:
Fixing linters erros

Fixing a little errors

BenMukebo commented on styles.css:
```diff
@@ -102,3 +102,227 @@ header a {
+.tag-element:hover {
+  background-color: #36b37f;
+}
+
+.see-project-btn {
+  width: 122px;
```
- [x] I salute your work of art, you have accomplished a great work, there are a few things you will need to work on before you go to the next step, let's take a look at this below and make all as required, please 😄 
-  Kindly implement the button actions as `hover, disable, enable, active` as per the project requirement, see this image below ⏬ 

<img width="482" alt="actio" src="https://user-images.githubusercontent.com/79058364/159062435-4f064e9f-c46f-4ae7-af48-1f2cf290cd50.png">
 
- Don't you think this color does not match the original design? alright, please take a look closer at it and make the change 😄 
### YOUR DESIGN
<img width="309" alt="color" src="https://user-images.githubusercontent.com/79058364/159063055-8f93971b-1db9-45f6-b9f9-e174ee608f26.png">

###  FIGMA

<img width="243" alt="gik" src="https://user-images.githubusercontent.com/79058364/159063123-1240d17d-d086-4a81-98eb-d35eef9ea2f4.png">

- If you take a look on line 20 in your HTML, I think you left this `>` there and now it displays to your design 😞, please remove it. 
<img width="121" alt="png" src="https://user-images.githubusercontent.com/79058364/159063674-979ffc7c-960b-4a7b-9e17-ac34722f5d59.png">

BenMukebo reviewed (changes requested):
### Hi *** @rega1237, ***

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media3.giphy.com/media/wA6aYAO3jH6RmUIBYs/giphy.gif"/>

### To highlight 💪🏻

- No linter errors. ✔️ 
- Good commit messages ✔️ 
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rega1237 committed:
Add code rewiev changes

Add hover box-shadow to buttons
Add action style to buttons

Remove > on 3 lines menu

Change font weight text-work-title

rega1237 committed:
fixing linters error

Fix linters errors
